pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "bitwarden_workflow_linter"
+dynamic = ["version"]
+authors = ["Bitwarden Inc"]
+description = "Custom GitHub Action Workflow Linter"
+readme = "README.md"
+requires-python = ">=3.11"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
```

Comment:
🎨 Given this, you can reorganize the license files too. That can be saved for a later date though.

Reply:
Can you expand on what you mean by reorganizing the license files? Can we remove the `./bitwarden_license` directory and the license files that won't pertain to this project?

Reply:
Correct. You'll end up with one `LICENSE.txt`.

Reply:
I reorganized down to the single `LICENSE.txt` file to be our GPL one.